.s2i/environment

```diff
@@ -0,0 +1 @@
+APP_MODULE=datanommer.app:app
```

Comment:
This module does not exist. According to [the documentation](https://github.com/sclorg/s2i-python-container/tree/master/3.9) it's only for running webapps with Gunicorn. In our case we want to run the `fedora-messaging consume` command so I think you'll have to provide an `app.sh` script.

Reply:
Ok, I'll try to check this.